policyengine_us/variables/gov/states/mt/tax/income/deductions/itemized/mt_itemized_deductions_joint.py

```diff
@@ -0,0 +1,38 @@
+from policyengine_us.model_api import *
+
+
+class mt_itemized_deductions_joint(Variable):
+    value_type = float
+    entity = Person
+    label = (
+        "Montana itemized deductions when married couples are filing jointly"
+    )
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf#page=7"
+        "https://law.justia.com/codes/montana/2022/title-15/chapter-30/part-21/section-15-30-2131/"
+        # MT Code § 15-30-2131 (2022) (1)
+    )
+    defined_for = StateCode.MT
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.irs.deductions
+        charitable_deduction = person.tax_unit("charitable_deduction", period)
```

Comment:
make this a single `add` call from a list parameter, can use comments to distinguish groups. for the individual one you can then subtract half the charitable deduction

Reply:
would a list facilitate the fact the we should only assign values to the head and spouse?

Reply:
I was thinking it could centralize at least the three common deductions in the `add` but it's fine either way

Reply:
I will file an issue to consolidate but for clarity I think this might be easier for now